src/content/developers/docs/programming-languages/ruby/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Ethereum for Ruby Developers
+description: Learn how to develop for Ethereum using Ruby-based projects and tooling.
+lang: en
+sidebar: true
+incomplete: true
```

Comment:
Also, you've marked this as incomplete in the frontmatter. This displays a banner at the top of the page and a call to action at the bottom of the page. I'll default to you as the expert on the Ruby Ethereum ecosystem here — would you say it is accurate to call this page incomplete?

<img width="1210" alt="Screenshot 2022-02-22 at 11 33 12" src="https://user-images.githubusercontent.com/62268199/155124224-4263bf09-8c1f-48dc-ab50-9dc9533a0e10.png">

<img width="853" alt="Screenshot 2022-02-22 at 11 34 29" src="https://user-images.githubusercontent.com/62268199/155124512-48e5d03b-67ee-470d-a2cf-954d043bfe8a.png">

Reply:
No, this page is fairly complete now. :-)